Parsers/ASimAuthentication/ProductParsers/AADSTS.yaml

```diff
@@ -0,0 +1,46 @@
+LookupTable:
+  Title: AAD STS Error Codes
+  Version: '0.0'
+  LastUpdated: June 20, 2021
+Product:
+  Name: Azure Active Directory
+References:
+- Title: Using functions
+  Link: https://docs.microsoft.com/azure/azure-monitor/log-query/function
+- Title: Azure AD Authentication and authorization error codes
+  Link: https://docs.microsoft.com/azure/active-directory/develop/reference-aadsts-error-codes
+Description: |
+      This query creates a table for conversion of AAD STS Error Codes from a numeric value to a descriptive error message.
+LookupTableName: AADSTSErrorCodes
+LookupTableQuery: |
```

Comment:
This will require maintenance, do we have a way to source this automatically for additions/changes/removals?

Reply:
Unfortunately not :-(. So many normalization lookups are heavy research. In essence, we create this table.